packages/react-ai/src/components/AIConversation/AIConversationProvider.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+import { ElementsProvider } from '@aws-amplify/ui-react-core/elements';
+
+import { AIConversationInput, AIConversationProps } from './types';
+import { defaultAIConversationDisplayTextEn } from './displayText';
+import {
+  ConversationDisplayTextProvider,
+  SuggestedPromptProvider,
+  ConversationInputContextProvider,
+  AvatarsProvider,
+  ActionsProvider,
+  MessageVariantProvider,
+  MessagesProvider,
+  ControlsProvider,
+  LoadingContextProvider,
+  ResponseComponentsProvider,
+  SendMessageContextProvider,
+} from './context';
+import { AttachmentProvider } from './context/AttachmentContext';
+
+export const AIConversationProvider = ({
+  elements,
+  actions,
+  suggestedPrompts,
+  responseComponents,
+  variant,
+  controls,
+  displayText,
+  allowAttachments,
+  messages,
+  handleSendMessage,
+  avatars,
+  isLoading,
+  children,
+}: AIConversationInput &
+  AIConversationProps & { children?: React.ReactNode }): React.JSX.Element => {
```

Comment:
Let's define a single interface for the `props`:
```
interface ProviderProps extends AIConversationInput, AIConversationProps {
  children?: React.ReactNode
}
```
Will help ensure that there are no unintentional type overrides through intersection